code/game/machinery/camera/camera.dm

```diff
@@ -65,6 +75,10 @@
 	qdel(wires)
 	wires = null
 	cameranet.removeCamera(src) //Will handle removal from the camera network and the chunks, so we don't need to worry about that
+	cameranet.cameras -= src

```

Comment:
removeCamera does something entirely different that more or less lets the cameranet know "hey, there's been a change in a camera chunk"--it doesn't really do anything other than that.